tests/components/items/item_label.spec.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import ShallowRenderer from "react-test-renderer/shallow";
+
+import ItemLabel from "../../../src/components/items/item_label";
+
+describe("ItemLabel", () => {
+  test("default snapshot", () => {
+    const renderer = new ShallowRenderer();
+    const tree = renderer.render(<ItemLabel />);
+    expect(tree).toMatchSnapshot();
+  });
+
+  test("snapshot with label text", () => {
+    const label = "hi";
+    const renderer = new ShallowRenderer();
+    const tree = renderer.render(<ItemLabel label={label} />);
```

Comment:
can we have a snapshot with overflow and without?
Seems like the title is there 100% of the time based on the current snapshot